src/tests/models/manager.js

```diff
@@ -32,6 +33,10 @@ async function insertTestBenchMark(benchMarkRawData, testId) {
     await database.insertTestBenchMark(testId, dataParse);
     return { benchmark_data: benchMarkRawData };
 }
+async function getBenchmark(testId) {
+    const res = await database.getTestBenchMark(testId);
```

Comment:
1.better rename res=> benchmark
2.in some places in the code, it's Benchmark and some BenchMark
better rename all to Benchmark.
3. in case it's not found better throw exception with 404

Reply:
I don't think 404 is the right way, test id exists, I'm asking for related data to this test id, it can be empty result as far as I think.  

the other ones fixed